README.md

```diff
@@ -738,6 +738,23 @@ configuration file, please issue a rename like the following
 cp .pgenv.conf .pgenv.default.conf
 ```
 
+Please note also that, since version `1.2.1` [811ba05], all the configuration files
+have been moved into the `config` subdirectory, so to keep them in a single place.
+In order to "migrate" your existing configuration, you have to manually
+copy all the `.pgenv.*.conf` files into the `config` subdirectory.
+The best way to quickly migrate your configuration files to the new
+naming scheme, is running a small shell loop as the following one:
+
+
+```
+for f in .pgenv.*.conf; do
+  F=$( echo $f | sed 's/\.pgenv\.//' ); mv $f config/$F;
+done
+```
```

Comment:
Maybe we should add a command that just does this.

Reply:
I've implemented `config migrate` for that purpose.
Should it also migrate the configuration files to the array-based syntax using this?